js/main.js

```diff
@@ -1,14 +1,60 @@
+
 var clickHereButton = document.querySelector('.click-here-btn');
 clickHereButton.addEventListener('click', generateWord);
 function generateWord(event) {
   getNewWord();
+  goToNewWord();
+  var word = data.currentRandomCard.word;
+  var definition = data.currentRandomCard.definition;
+  var cardGenerated = {
+    word: word,
+    definition: definition,
+    wordId: data.nextWordId++
+  };
+
+  data.allCards.push(cardGenerated);
```

Comment:
Why are you saving all cards? I think you'll only need to save the words the user actually chooses to save